src/views/order.html

```diff
@@ -110,9 +110,31 @@
           </fieldset>
         </div>
       </div>
+    
+      <!--Free Usage-->
+      <div id="free" ng-show="orderModel.site != null && orderModel.env != null && orderModel.env != 'google'" style="margin-top:30px;">
+          <fieldset class="form-group">
+            <legend>Academic Use</legend>
+            <p>Web Platforms is free for use within official University of Michigan courses. Instances will be deleted six weeks after the end of the semester in which they are used.</p>
+            
+            <label for="free-input">Will your site/application be used to support a registered University course?</label>
+             <div class="form-check">
+                <label class="form-check-label">
+                    <input type="checkbox" ng-model="orderModel.free" class="form-check-input">
+                    Yes
+                </label>
+              </div>
+              <div id="course" ng-show="orderModel.free">
+                <input type="text" ng-model="orderModel.course" class="form-control" placeholder="Enter course name" style="width:30%;">
+              </div>
+              <div id="term" ng-show="orderModel.free">
+                <input type="text" ng-model="orderModel.term" class="form-control" placeholder="Enter term" style="width:30%;">
+              </div>
+          </fieldset>
+      </div>
 
       <!--Database Options-->
-      <div id="db" ng-show="orderModel.site ==='app' && orderModel.env">
+      <div id="db" ng-show="orderModel.site ==='app' && orderModel.env != null">
```

Comment:
@itarpeh 
Better always use !== so that we are certain of checking both value and type